### PR TITLE
fix android external links

### PR DESCRIPTION
**Pull request by toddmedema**
Closes https://github.com/ExpeditionRPG/expedition-app/issues/636

### CODE REVIEW

File: app/components/FeaturedQuests.tsx [hunk 1]
```diff
@@ -48,11 +49,11 @@ const FeaturedQuests = (props: FeaturedQuestsProps): JSX.Element => {
           <div className="title"><img className="inline_icon" src="images/roll_small.svg"/>Tools</div>
         </div>
       </Button>
-      <a href="#" onClick={() => window.open('https://expeditiongame.com/store?utm_source=app', '_system')} className="linkButton"><Button>
+      <Button onTouchTap={()=>openWindow('https://expeditiongame.com/store?utm_source=app')} remoteID="tools">
```
Comment (smartin015) [inline app/components/FeaturedQuests.tsx]: Shouldn't `remoteID="shop"`? Alternatively if you leave this off, we won't display tap ripples from other clients (which kind of makes sense).

File: package.json [hunk 2]
```diff
@@ -9,7 +9,8 @@
     "test": "mocha meta_tests.js && karma start --single-run --browsers PhantomJS",
     "watch:test": "karma start",
     "dev": "webpack-dev-server --progress",
-    "build": "webpack --config ./webpack.dist.config.js --progress"
+    "build": "webpack --config ./webpack.dist.config.js --progress",
+    "build-all": "npm run build && cordova build android && cordova build ios"
```
Comment (smartin015) [inline package.json]: I'm not on a mac, so this won't work for me :(

File: app/components/FeaturedQuests.tsx (hunk 1 above)
Comment (toddmedema) [reply app/components/FeaturedQuests.tsx]: Good call, rm'd

File: package.json (hunk 2 above)
Comment (toddmedema) [reply package.json]: Yep, that's why there's still the normal `build` :) PITA to type all of that out when I'm testing something against all the devices we support.

**Pull request merged**